### ISSUE TITLE
"User menu" has two borders after closing and opening app

### ISSUE THREAD

**Issue by LynHyper** [vector-im/element-web#20367]
### Steps to reproduce

1. Start Element Desktop
2. Click a random room under "Rooms" or "People"
3. Close Element (to it's minimised state)
4. Open Element from the system tray
5. See the "User menu" icon in the top-left be highlighted with an inner yellow border and an outer white border (See image below)

![usermenuiconbug](https://user-images.githubusercontent.com/86165318/147603591-2a6d755f-4e82-4df2-80c0-dc618fe02ee1.png)

#### What did you expect?

The "User menu" icon shouldn't be highlighted from closing and opening the app.

#### What happened instead?

The "User menu" icon was highlighted with an inner yellow border and an outer white border. Some notes about the "User menu" borders is that:

1. The white border is the same thickness as the border of the "All rooms" icon underneath it, but, it's underneath the yellow border hence why it appears thinner.
2. The yellow border on top is actually thinner than normal.
3. Both borders seem to have smaller border-radius values than normal, they also both appear to be underneath the user icon as well.
4. Both borders disappear as soon as you click inside or outside of Element unlike the "All rooms" icon's border.

### Operating system

Arch Linux

### Application version

Element version: 1.9.8, Olm version: 3.2.8

### How did you install the app?

FlatHub

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by SimonBrandner** [comment]
This is caused by your browser/Electron, it means the element is selected, this is useful for keyboard navigation and you can usually trigger this with pressing tab

**Comment by kittykat** [comment]
I've marked as release blocker because it's a visually impactful bug which I can reproduce reliably on the same system as the reporter.

**Comment by t3chguy** [comment]
Does not seem to affect macOS

**Comment by t3chguy** [comment]
This one is proving a pain to investigate as my Parallels network stack has decided to throw a tantrum :((

**Comment by t3chguy** [comment]
Cannot reproduce on Ubuntu either

https://user-images.githubusercontent.com/2403652/148529991-e78811e8-6fe5-48a4-b9fa-6ec587cbc5eb.mov

**Comment by gsouquet** [comment]
> Cannot reproduce on Ubuntu either

Same here, installed an Ubuntu VM and could not reproduce

**Comment by t3chguy** [comment]
Cannot reproduce on Windows either, the occurrence seems wrong.

**Comment by LynHyper** [comment]
If it helps, I was using XFCE4 as my DE when this visual bug occured, it might be specific to it; I haven't tried this with other desktop environments.

**Comment by kittykat** [comment]
I'm on GNOME with Wayland, on Arch.

**Comment by t3chguy** [comment]
Worth re-testing after https://github.com/vector-im/element-desktop/pull/299 lands

**Comment by LynHyper** [comment]
Just to update, I'm using `Element version: 1.9.9, Olm version: 3.2.8` and I can still reproduce the error.

**Comment by LynHyper** [comment]
I can confirm using Element version: 1.10.0, Olm version: 3.2.8 that this bug is no longer happening. I'll close this issue.
